### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by cameronf**
So, the gsub that was here was replacing the e-27 in scientific notation with e@27, and then the lexer couldn't parse that.

I did a bunch of research and the things we were trying to parse by adding that - namely `1%-1:1000` - we weren't actually parsing it correctly - we were parsing that as a 11000/10000mg/ml concentration, when that isn't what that is, it is a multichem drug with no sane simplified concentration (see #doctor-check for more details)

So I got aggressive and removed the gsub, and the multichem handling all together, as well as adding a regex to handle scientific notation.

I did builds of both bcs and phi-service, and they seemed to pass - doing a svalbard build now as a final test.

Lastly - @rossreed if you had a sec to do whatever needs to be done to this repo to prevent it from using the gems.kitcheck.com server, that wouldn't hurt....

### CODE REVIEW

File: lib/unit/lexer_definition.rex [hunk 1]
```diff
@@ -3,20 +3,20 @@ option
   ignorecase
 macro
   BLANK       [\ \t]+
+  SCIENTIFIC  -?\d\.?\d*[Ee][\+\-]?\d+
```
Comment (wimpeemc) [inline lib/unit/lexer_definition.rex]: first digit match should be `\d*` or it'll fail on `-.5e1mg` and `11.5e1mg`

Comment (cameronf) [reply lib/unit/lexer_definition.rex]: I thought about that, but then I argued that neither of those are truly scientific notation. I'm 55/45 though so if you think it's better to handle I'll update

Comment (cameronf) [reply lib/unit/lexer_definition.rex]: maybe \d+ at a minimum?  Otherwise won't it effectively match E5mg which would blow up.  Maybe that's OK too....

Comment (wimpeemc) [reply lib/unit/lexer_definition.rex]: if it's gotta be a single regex i'd kinda lean towards
```
-?[\d.]+[Ee][+-]?\d+
```
which technically allows for multiple dots where it shouldn't, but it also catches `12.e-5`, `12.3e-5`, and `-.1e2` which the original version didn't. if overmatching is worse than undermatching, going with `\d+` sounds reasonable too

**Comment by cameronf** [comment]
@wimpeemc - updated above

**Pull request merged**